dashboard/db/migrate/20170427233825_add_bonus_to_script_level.rb

```diff
@@ -0,0 +1,5 @@
+class AddBonusToScriptLevel < ActiveRecord::Migration[5.0]
+  def change
+    add_column :script_levels, :bonus, :boolean
```

Comment:
Ideally yes, but I'm following the pattern of the `assessment` & `named_level` columns.